src/frontend/komponenter/Fagsak/Simulering/Simulering.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+
+import { useHistory } from 'react-router';
+import styled from 'styled-components';
+
+import Alertstripe from 'nav-frontend-alertstriper';
+
+import { useHttp } from '@navikt/familie-http';
+
+import { aktivVedtakPåBehandling } from '../../../api/fagsak';
+import { IBehandling } from '../../../typer/behandling';
+import { IFagsak } from '../../../typer/fagsak';
+import Skjemasteg from '../../Felleskomponenter/Skjemasteg/Skjemasteg';
+
+interface ISimuleringProps {
+    fagsak: IFagsak;
+    åpenBehandling: IBehandling;
+}
+
+const StyledAlertstripe = styled(Alertstripe)`
+    margin-bottom: 2rem;
+`;
+
+const Simulering: React.FunctionComponent<ISimuleringProps> = ({ åpenBehandling, fagsak }) => {
+    const { request } = useHttp();
+    const history = useHistory();
+    const aktivtVedtak = aktivVedtakPåBehandling(åpenBehandling);
+
+    const nesteOnClick = async () => {
```

Comment:
Hva er ønsket effekt av bekreft? Er det lagring av tilbakekrevingsvalgene?

Ville uansett skrevet om til å bruke then/catch slik at vi fanger opp eventuelle feil og presenterer disse for bruker :)